src/sentry/replays/usecases/ingest/dom_index.py

```diff
@@ -128,7 +128,11 @@ def log_canvas_size(
     events: list[dict[str, Any]],
 ) -> None:
     for event in events:
-        if event.get("type") == 3 and event.get("data", {}).get("source") == 9:
+        if (
+            event.get("type") == 3
+            and event.get("data", {}).get("source") == 9
+            and random.randint(0, 499) < 1
```

Comment:
any specific reason this one is `499` and the other is `500`?

Reply:
ah, no reason 😆